cg/cli/backup.py

```diff
@@ -34,6 +40,42 @@ def backup(context: CGConfig):
     pass
 
 
+@backup.command("flow-cells")
+@DRY_RUN
+@click.pass_obj
+def backup_flow_cells(context: CGConfig, dry_run: bool):
+    """Backup flow cells."""
+    pdc_api = PdcAPI(binary_path=context.pdc.binary_path, dry_run=dry_run)
+    status_db: Store = context.status_db
+    for flow_cell in get_flow_cells_from_path(flow_cells_dir=Path(context.flow_cells_dir)):
+        db_flow_cell: Optional[Flowcell] = status_db.get_flow_cell_by_name(
+            flow_cell_name=flow_cell.id
+        )
+        flow_cell_encryption_api = FlowCellEncryptionAPI(
```

Comment:
Not a fan of instantiating this per flow cell. Could be created in the context instead to remove setup logic from the cli command. Out of scope here

Reply:
I agree with @seallard the `FlowCellEncryptionAPI` could need a re-work. But out of scope...